runtime/near-vm-runner/src/logic/gas_counter.rs

```diff
@@ -11,15 +11,15 @@ use std::collections::HashMap;
 
 #[inline]
 pub fn with_ext_cost_counter(f: impl FnOnce(&mut HashMap<ExtCosts, u64>)) {
-    #[cfg(feature = "costs_counting")]
+    #[cfg(any(test, feature = "costs_counting"))]
```

Comment:
This file has the changes that fix parts of the near-vm-runner test suite not actually being checked with `-p near-vm-runner` before